marshmallow/__init__.py

```diff
@@ -14,7 +14,7 @@
 from marshmallow.utils import pprint, missing
 from marshmallow.exceptions import ValidationError
 
-__version__ = '2.11.1'
+__version__ = '2.11.2'
```

Comment:
No need to bump the version; I will do this before I release. 

This is technically a feature rather than a bugfix, so it will go in 2.12.0. Would you mind reverting this change?